src/bits/bit_field_vec.rs

```diff
@@ -93,20 +93,102 @@ fn mask<W: Word>(bit_width: usize) -> W {
 }
 
 impl<W: Word> BitFieldVec<W, Vec<W>> {
-    /// Create a new zero-initialized vector of given bit width and length.
-    pub fn new(bit_width: usize, len: usize) -> Self {
-        // We need at least one word to handle the case of bit width zero.
+    /// Returns a new BitFieldVec filled with `word`
+    ///
+    /// # Arguments
+    /// * `bit_width` - The bit width of the values stored in the vector.
+    /// * `len` - The length of the vector.
+    /// * `word` - The value to fill the vector with.
+    ///
+    /// # Example
```

Comment:
Note that as per guidelines https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#detailed-design the section name is "Examples", not "Example", even if it's one.

Reply:
Resolved